src/secret_sharing/replicated/malicious/additive_share.rs

```diff
@@ -23,16 +23,37 @@ use std::{
 use typenum::Unsigned;
 
 #[derive(Clone, PartialEq, Eq)]
-pub struct AdditiveShare<V: SharedValue> {
+pub struct AdditiveShare<V: SharedValue + ExtendableField> {
     x: SemiHonestAdditiveShare<V>,
-    rx: SemiHonestAdditiveShare<V>,
+    rx: SemiHonestAdditiveShare<V::LargeFieldType>,
 }
 
-impl<V: SharedValue> SecretSharing<V> for AdditiveShare<V> {
+pub trait ExtendableField {
+    type LargeFieldType: Field;
```

Comment:
I think we should call this...

```suggestion
    type InducedField: Field;
```

or

```suggestion
    type ExtendedField: Field;
```

Reply:
ok, I'll go with "ExtendedField"